Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -889,35 +889,35 @@ namespace OpenInfraPlatform {
 					//		Position: OPTIONAL IfcAxis2Placement;
 					//	END_ENTITY;
 					// **************************************************************************************************************************
-					// IfcSpiral SUBTYPE OF IfcBlossCurve
+					// IfcBlossCurve SUBTYPE OF IfcSpiral
 					if (spiral.isOfType<typename IfcEntityTypesT::IfcBlossCurve>())
 					{
 						return convertIfcSpiral(spiral.as<typename IfcEntityTypesT::IfcBlossCurve>(),
 							targetVec, segmentStartPoints, trim1Vec, trim2Vec, senseAgreement, trimmingPreference);

```

Comment:
I'd expect the function be called `convertIfcBlossCurve`

Reply:
Resolved e0dac45